code/modules/medical/cloning.dm

```diff
@@ -520,32 +520,45 @@
 		else
 	return
 
-/obj/machinery/cloning/clonepod/AltClick()
-	var/visible_message = FALSE
+/obj/machinery/cloning/clonepod/MouseDrop_T(obj/item/weapon/reagent_containers/food/snacks/meat/M, mob/living/user)
 	var/busy = FALSE
+	var/visible_message = FALSE
+
+	if(!istype(M))
+		return
+
+	if(issilicon(user))
+		return //*buzz
+
+	if(!Adjacent(user) || !user.Adjacent(src) || M.loc == user || !isturf(M.loc) || !isturf(user.loc) || user.loc==null)
+		return
+
+	if(user.incapacitated() || user.lying)
+		return
+
+	if(stat & (NOPOWER|BROKEN))
+		return
+
 	if(!busy)
 		busy = TRUE
-		if(!issilicon(usr) && !usr.incapacitated() && Adjacent(usr) && !(stat & (NOPOWER|BROKEN)))
-			for(var/obj/item/weapon/reagent_containers/food/snacks/meat/meat in range(1, src))
-				biomass += BIOMASS_CHUNK
-				qdel(meat)
-				visible_message = TRUE // Prevent chatspam when multiple meat are near
-
-			if(visible_message)
-				playsound(get_turf(src), 'sound/machines/juicer.ogg', 30, 1)
-				visible_message("<span class = 'notice'>[src] sucks in and processes the nearby biomass.</span>")
+		for(var/obj/item/weapon/reagent_containers/food/snacks/meat/meat in M.loc)
```

Comment:
I think you are accepting any kind of snack here.

Reply:
any kind of meat, like the manual processing.